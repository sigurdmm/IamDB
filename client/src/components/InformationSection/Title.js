import PropTypes from 'prop-types';
import React from 'react';

const Title = ({ title }) => <div className='title__container'><h2>{title}</h2></div>;

Title.propTypes = {
  title: PropTypes.string.isRequired,
};

export default Title;
