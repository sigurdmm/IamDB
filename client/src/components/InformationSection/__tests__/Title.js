import React from 'react';
import { render } from 'enzyme/build';
import Title from '../Title';

describe('<Title/>', () => {
  it('should render title', () => {
    const wrapper = render(<Title title='media title'/>);

    expect(wrapper).toMatchSnapshot();
  });
});
