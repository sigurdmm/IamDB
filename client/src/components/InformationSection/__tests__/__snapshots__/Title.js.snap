// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Title/> should render title 1`] = `
<div
  class="title"
>
  <h2>
    media title
  </h2>
</div>
`;
